book/src/getting-started/5-cmake-integration.md

```diff
@@ -0,0 +1,153 @@
+<!--
+SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+
+SPDX-License-Identifier: MIT OR Apache-2.0
+-->
+
+# Building with CMake
+
+```diff,ignore
+- Disclaimer: The CMake integration for CXX-Qt is still WIP.
+- The current state is far from optimal and will likely improve
+- a lot in the future, so don't be discouraged by anything in
+- this chapter.
+- Contributions are also welcome.
+```
+
+Before we can get started on building Qt with CMake, we should first make our Cargo build ready for it.
+If you've generated your project with the `cargo new --lib` command, your Cargo.toml likely looks something like this:
+```toml,ignore
+[package]
+name = "cxx-qt-getting-started"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+```
+
+We'll have to do multiple things:
+- Instruct cargo to create a static lib with a defined name for CMake to link against ("rust").
+- Add `cxx`, `cxx-qt`, as well as `cxx-qt-lib` as dependencies.
+- Add `clang-format` and `cxx-qt-build` as build-dependencies.
+
+In the end, your `Cargo.toml` should look like this:
+```toml,ignore
+[package]
+name = "cxx-qt-getting-started"
+version = "0.1.0"
+edition = "2021"
+
+# Don't touch this, the CMake file expects a static lib with a specific name
```

Comment:
obviously they don't have to put this comment, this was more a note for ourselves, but maybe still useful to have :-)

Reply:
Done